typescript/rebalancer/src/utils/files.ts

```diff
@@ -0,0 +1,266 @@
+import { input } from '@inquirer/prompts';
```

Comment:
This file really makes me think that we would benefit from a `utils/fs` package/export cause the cli and infra already use different sets of filesystem based utils